lib/uo_cli.rb

```diff
@@ -406,19 +399,36 @@ def self.run_func
 
       feature_file = URBANopt::GeoJSON::GeoFile.from_file(featurefile)
       if @opthash.subopts[:reopt] == true || @opthash.subopts[:reopt_scenario] == true || @opthash.subopts[:reopt_feature] == true
```

Comment:
does "@opthash.subopts[:reopt] == true" here correspond to a valid state anymore?

Reply:
Nope! Good catch.